.github/workflows/ruff_linter.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: ["3.9"]
+        python-version: ["3.11"]
```

Comment:
so generally we want to make sure ao works on python versions 3.9+ so what's teh goal of this change

Reply:
3.11 is newer but your context makes sense. I'll revert.